### ISSUE TITLE
Parse search parameter

### ISSUE THREAD

**Issue by fregante** [refined-github/shorten-repo-url#25]
Issue search links are messy:

```
https://github.com/sindresorhus/refined-github/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc
```

and are currently shortened to:

```
sindresorhus/refined-github/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc
```

I think they can be shortened to

```
sindresorhus/refined-github/issues (is:open sort:updated-desc)
```

Other search parameters, if any, should be left untouched

```
sindresorhus/refined-github/issues?foo=bar (is:open sort:updated-desc)
```

Failing tests in #24

**Comment by fregante** [comment]
Fixed by #24

**Comment by fregante** [comment]
This should also apply to global search URLs

```
https://github.com/issues?q=is%3Aissue+is%3Aopen+author%3Afregante+archived%3Afalse+sort%3Aupdated-desc
```

```
[issues (is:issue is:open author:fregante archived:false sort:updated-desc)](https://github.com/issues?q=is%3Aissue+is%3Aopen+author%3Afregante+archived%3Afalse+sort%3Aupdated-desc)
```
